src/ng/components/selection/indeterminate.js

```diff
@@ -0,0 +1,28 @@
+import Directive from '../directive';
+import {INDETERMINATE_NAME} from '../../../definition';
+
+class Indeterminate extends Directive(INDETERMINATE_NAME) {
+	constructor($scope, $element, $attrs) {
+		super();
+
+		this.$scope = $scope;
+		this.$element = $element;
+		this.$attrs = $attrs;
+	}
+
+	onInit() {
+		this.$scope.$watch(this.$attrs[INDETERMINATE_NAME], (newVal) => {
```

Comment:
$observe fires only if I change binding expression itself and newVal always has the same value "$selection.indeterminate()" in my case. Rather than $watch callback fires each time function result $selection.indeterminate() changes)